README.md

```diff
@@ -348,6 +360,13 @@ Or do the same with an online JSON file:
 $ curl -sL "http://git.io/aorKgQ" | jsesc --json --object > data-escaped.json
 ```
 
+For lowercase hexadecimal values, use the `-l`/`--lowercase-hex` option:
```

Comment:
This section can be removed — only a few examples of common usage are listed here.